Sources/TokamakDemo/TokamakDemo.swift

```diff
@@ -125,6 +125,12 @@ var links: [NavItem] {
     gridDemo,
     redactDemo,
   ]
+
+  #if os(WASI)
+  result.append(NavItem("DOM manipulation", destination: DOMRefDemo()))
```

Comment:
Could you extract this like some of the other demos and replace it with a `NavItem(unavailable: "DOM manipulation")` on native?

Reply:
Great point!